setup.py

```diff
@@ -22,7 +22,7 @@
         'twilio>=6.3.0,<7',
         'Django>=1.8,<1.9;python_version=="3.3"',
         'Django>=1.8,<2;python_version<"3.4"',
-        'Django>=1.8,<2.1;python_version>="3.4"',
+        'Django>=1.8,<=2.1;python_version>="3.4"',
```

Comment:
I don't think this does what you're intending. This is the same as saying `<=2.1.0`, so it won't work with security releases. If you're wanting to relax it to allow the latest `2.1.X`, you probably want `<2.2`.

Reply:
Totally right! Thank you. I definitely meant `<2.2`